common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -0,0 +1,146 @@
+.. _common-imu-notch-filtering:
+
+============================================================================
```

Comment:
need to add [copywiki destination="copter"]  since this is only site that references this document and creates wiki build errors otherwise for the other sites

Reply:
Fixed